pkg/defaults/defaults.go

```diff
@@ -54,7 +54,7 @@ const (
 
 	//tags, versions, repos
 	DefaultEVETag               = "10.11.0" // DefaultEVETag tag for EVE image
-	DefaultAdamTag              = "0.0.43"
+	DefaultAdamTag              = "0.0.49"
```

Comment:
Curious (sorry for off-topic), are we really need to update adam tag with https://github.com/lf-edge/adam/pull/113 merged (and without changes of adam internal endpoints)?

Reply:
You are right, this works just fine with the older Adam. I have removed this line change from the PR.

Reply:
(still keeping as draft until EVE PR is merged)